ops/testing.py

```diff
@@ -1963,6 +2071,7 @@ def __init__(self, unit_name: str, meta: charm.CharmMeta, config: '_RawConfig'):
         self._opened_ports: Set[model.Port] = set()
         self._networks: Dict[Tuple[Optional[str], Optional[int]], _NetworkDict] = {}
         self._reboot_count = 0
+        self._operation = _Operation(ActionOutput([], {}), {})
```

Comment:
I think `_Operation` is not descriptive enough a name. How about `_ActionContext` or `_RunningAction`?

Reply:
That gave me pause too. The rationale is that Juju calls these things "operations": `juju operations`, `juju show-operation`, etc. However, in this case I'm inclined to agree with Pietro that something like `_ActionInfo` is better. We don't have the concept of "operation" in ops yet, and by itself it seems too vague.

Reply:
Ok, fine with me. I indeed just pulled the name from Juju (I originally had a bunch of individual attributes and it seemed noisy so needed a name to bundle them under). If we do ever add "list-operations" etc (I couldn't think of any use case for that) then it's simple enough to rename the private class at the same time as expanding it to have everything else it would need.